cmd/mountv1.go

```diff
@@ -118,6 +118,8 @@ var useAttrCache bool
 var useStream bool
 var useFileCache bool = true
 var convertConfigOnly bool
+var enableGen1 bool
```

Comment:
one consideration, maybe in azstorage if a user mistakenly passes gen1 or something and uses the mount command, can we print out an error message that says use the mount gen1 command or something?

Reply:
In mount also we want to support gen1 mount not just using mountv1

Reply:
mountgen1 is subcommand that can be used directly with v2 binary to do gen1 mount

Reply:
So is mount doing the transform for the user from v2 to gen1 style config? and mountgen1 just takes in directly the gen1 configurations?

Reply:
In V1 when you mount we generate json file and internally execute the gen1fuse binary. So we are following the same model just that in v2 instead of a cli flag we have a new subcommand which converts config to json and starts gen1fuse binary. mounv1 command is also suppose to support the flag that v1 had, so in mountv1 we convert v1 config to v2 and if enable-gen1 flag is there then we execute the newly added subcommand of v2 which internally will fire gen1fuse.